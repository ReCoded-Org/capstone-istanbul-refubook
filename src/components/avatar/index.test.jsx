import React from 'react';
import Avatar from './';
import renderer from 'react-test-renderer';
import editIcon from '../../assets/editIcon.svg';

const userInfo = {
  id: 1,
  name: 'Jonathan Reinink',
  avatar: 'https://via.placeholder.com/150',
  profile: '',
};

it('Avatar page renders correctly', () => {
  const tree = renderer
    .create(<Avatar user={userInfo} buttonIcon={editIcon} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
