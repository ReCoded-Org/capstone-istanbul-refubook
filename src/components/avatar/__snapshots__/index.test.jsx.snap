// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar page renders correctly 1`] = `
<div
  className="flex flex-col justify-center mt-10 mx-auto"
>
  <div
    className="flex relative w-24 h-24 bg-orange-500 justify-center items-center m-1 mr-2 text-xl rounded-full text-white"
  >
    <img
      alt="A"
      className="rounded-full"
      src="https://via.placeholder.com/150"
    />
    <button
      className="flex items-center justify-around p-0 w-8 h-8 bg-blue-400 rounded-full absolute bottom-0 right-0 hover:bg-blue-600 active:shadow-lg mouse shadow transition ease-in duration-200 focus:outline-none"
    >
      <img
        alt="edit Icon"
        className="w-4 h-4 inline-block fill-current text-blue-100"
        src="editIcon.svg"
      />
    </button>
  </div>
  <p>
    Jonathan Reinink
  </p>
</div>
`;
